airflow/cli/cli_parser.py

```diff
@@ -153,6 +153,12 @@ def __init__(self, flags=None, help=None, action=None, default=None, nargs=None,
     ),
     choices=tabulate_formats,
     default="fancy_grid")
+ARG_COLOR = Arg(
+    ('--color',),
+    help="do emit colored output (default: auto-detect)")
+ARG_NO_COLOR = Arg(
+    ('--no-color',),
+    help="do not emit colored output (default: auto-detect)")
```

Comment:
```suggestion
ARG_COLOR = Arg(
    ('--color',),
    help="do emit colored output (default: auto-detect)",
    action="store_true")
ARG_NO_COLOR = Arg(
    ('--no-color',),
    help="do not emit colored output (default: auto-detect)",
    action="store_false",
    dest="color")
```

And then should_use_colors is a bit simpler.